server_environment_ir_config_parameter/tests/test_server_environment_ircp.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2016-2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import UserError
+from odoo.tests import common
+from odoo.tools import convert_file
+from odoo.modules.module import get_resource_path
+
+
+class TestEnv(common.TransactionCase):
+
+    def setUp(self):
```

Comment:
Isn't setupclass just better?

Reply:
no, if you don't do anything :)
setUpClass must be used if you are creating several records that are used by each test.